charts/scoold/README.md

```
@@ -0,0 +1,54 @@
+
+```console
+$ helm repo add one-acre-fund https://one-acre-fund.github.io/oaf-public-charts
+$ helm install my-release one-acre-fund/scoold
+```
+
+## Values
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| affinity | object | `{}` |  |
+| applicationConf | string | `"scoold.env = production\nscoold.para_endpoint = \"https://paraio.com\"\nscoold.para_access_key = \"app:scoold\"\nscoold.para_secret_key = \"secret\"\n#scoold.password_auth_enabled = true\n#scoold.is_default_space_public = true\n####################################\n# Add more config properties here...\n####################################\n"` | See https://github.com/Erudika/scoold for more details |
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)